math/iteration_power.py

```diff
@@ -0,0 +1,35 @@
+def iteration_power(number: int, exp: int) -> int:
+    """
+    Perpangkatan dengan metode iteratif atau perulangan
+    rumus matematika: number^exp
+    >>> iteration_power(2, 5)
+    32
+    >>> iteration_power(100, 0)
+    1
+    >>> iteration_power(0, 100)
+    0
+    >>> iteration_power(1, 100)
+    1
+    """
+    result = 1
+    for i in range(exp):
```

Comment:
fungsi dari ``i`` tidak dipergunakan, untuk itu maka gunakan underscore ``_`` untuk membuat anonymous variabel
```suggestion
    for _ in range(exp):
```